include/simulation/Feedback.hpp

```diff
@@ -11,7 +12,7 @@ typedef struct SimulationError {
 
 typedef struct RobotControlFeedback {
     bool isTeamYellow;
-    std::vector<int> robotsThatHaveTheBall;
+    std::map<int, bool> robotIdHasBall;
```

Comment:
Is this map ever cleared? If a robot is taken out of the field, is it also taken out of the map?